packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -290,13 +302,30 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
+const inputNumberWidth = ( { size }: InputNumberProps ) => {
+	const sizes = {
+		default: space( 16 ),
+		'__unstable-large': space( 20 ),
+	};
+
+	if ( size === '__unstable-large' ) {
+		return css( {
+			width: `${ sizes[ '__unstable-large' ] }`,
+		} );
+	}
+
+	return css`
+		width: ${ sizes.default };
+	`;
+};
```

Comment:
We can simplify the logic in this function a bit:

```suggestion
const inputNumberWidth = ( { size }: InputNumberProps ) => {
	const sizes = {
		default: space( 16 ),
		'__unstable-large': space( 20 ),
	};

	return css`
		width: ${ sizes[ size ?? 'default' ] };
	`;
};
```

Reply:
Good call

Reply:
We can actually simplify this a little bit further, by re-using `NumberControl` prop types

```diff
diff --git a/packages/components/src/range-control/styles/range-control-styles.ts b/packages/components/src/range-control/styles/range-control-styles.ts
index e0da254e12..a1ed8852bf 100644
--- a/packages/components/src/range-control/styles/range-control-styles.ts
+++ b/packages/components/src/range-control/styles/range-control-styles.ts
@@ -10,9 +10,8 @@ import styled from '@emotion/styled';
 import NumberControl from '../../number-control';
 import { COLORS, reduceMotion, rtl } from '../../utils';
 import { space } from '../../ui/utils/space';
-
+import type { NumberControlProps } from '../../number-control/types';
 import type {
-	InputNumberProps,
 	RangeMarkProps,
 	RailProps,
 	ThumbProps,
@@ -302,20 +301,17 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
-const inputNumberWidth = ( { size }: InputNumberProps ) => {
+const inputNumberWidth = ( {
+	size = 'default',
+}: Pick< NumberControlProps, 'size' > ) => {
 	const sizes = {
+		small: space( 16 ),
 		default: space( 16 ),
 		'__unstable-large': space( 20 ),
 	};
 
-	if ( size === '__unstable-large' ) {
-		return css( {
-			width: `${ sizes[ '__unstable-large' ] }`,
-		} );
-	}
-
 	return css`
-		width: ${ sizes.default };
+		width: ${ sizes[ size ?? 'default' ] };
 	`;
 };
 
diff --git a/packages/components/src/range-control/types.ts b/packages/components/src/range-control/types.ts
index ffccd327e3..4f7c5571fd 100644
--- a/packages/components/src/range-control/types.ts
+++ b/packages/components/src/range-control/types.ts
@@ -277,12 +277,6 @@ export type TrackProps = {
 	trackColor: CSSProperties[ 'color' ];
 };
 
-export type Size = 'default' | '__unstable-large';
-
-export type InputNumberProps = {
-	size?: Size;
-};
-
 export type UseControlledRangeValueArgs = {
 	/**
 	 * The initial value.

```